internal/server/rule_manager.go

```diff
@@ -180,11 +180,30 @@ func deleteRule(name string) (result string) {
 }
 
 func startRule(name string) error {
+	return reRunRule(name)
+}
+
+// reRunRule rerun the rule from optimize to Open the operator in order to refresh the schema
+func reRunRule(name string) error {
 	rs, ok := registry.Load(name)
 	if !ok {
 		return fmt.Errorf("Rule %s is not found in registry, please check if it is created", name)
 	} else {
-		err := rs.Start()
+		r := rs.Rule
```

Comment:
Looks like you only want to re-construct the topo of the rule. It seems not need to change the rule registry? Just like updateRule, just need to updateTopo.

Reply:
updated.